app/helpers/podcasts_helper.rb

```diff
@@ -5,6 +5,23 @@ module PodcastsHelper
 
   RSS_LANGUAGE_CODES = %w[af sq eu be bg ca zh-cn zh-tw hr cs da nl nl-be nl-nl en en-au en-bz en-ca en-ie en-jm en-nz en-ph en-za en-tt en-gb en-us en-zw et fo fi fr fr-be fr-ca fr-fr fr-lu fr-mc fr-ch gl gd de de-at de-de de-li de-lu de-ch el haw hu is in ga it it-it it-ch ja ko mk no pl pt pt-br pt-pt ro ro-mo ro-ro ru ru-mo ru-ru sr sk sl es es-ar es-bo es-cl es-co es-cr es-do es-ec es-sv es-gt es-hn es-mx es-ni es-pa es-py es-pe es-pr es-es es-uy es-ve sv sv-fi sv-se tr uk]
 
+  def feed_description(feed, podcast)
+    desc = feed.description.present? ? feed.description : podcast.description
+    desc ||= ""
+    if podcast.has_apple_config?
+      desc = desc.truncate_bytes(4000, omission: "")
```

Comment:
We have [`Episode#description_with_default`](https://github.com/PRX/feeder.prx.org/blob/main/app/models/episode.rb#L313-L315) which is doing the work of the "other" side of the conditional branches in our validation. I.e. the part that's not encoded in Rails validation rules, but represents non-validated output that is transformed into validated output .  IIRC, Apple _requires_ a description on the episodes, which is why `description_with_default` was added (we don't have that validation on the episode currently?).

Going by the existing pattern,  I'd expect the fallback to `podcast.description` and the truncation of the bytes here to be in a similar `#description_with_default` type of method.

Reply:
Does a blank description seem problematic?  Another aspect of the existing `description_with_default` pattern.

Reply:
RSS spec requires a description tag, but not that there be a value. I see some other hosts require a desc, but generally have not.

Good news is I was able to get rid of truncation here as none of our feeds have a desc > 4k bytes, and instead i just left a helper method to handle choosing between the podcast or feed description override.